recipes/scxa-plots/meta.yaml

```diff
@@ -0,0 +1,40 @@
+{% set version = "0.0.1" %}
+
+package:
+  name: scxa-plots
+  version: {{ version }}
+
+source:
+  url: https://github.com/ebi-gene-expression-group/scxa-plots/archive/v{{ version }}.tar.gz 
+  sha256: 1aa824fcbb981d63ca1e329c3731c2b52d8d76777b082d6011ce668e1abec829 
+
+build:
+  number: 0
+  noarch: generic
+
+requirements:
+    run:
+        - bioconductor-dropletutils
+        - openblas
```

Comment:
why do you need openblas as runtime depdendeny?

Reply:
It was required when we first build this tool (I don't recall the error that prompted it to be added). Happy to take it out and reinstate later if required.